src/components/marker-chart/Canvas.js

```diff
@@ -10,7 +10,7 @@ import {
   type WithChartViewport,
 } from '../shared/chart/Viewport';
 import ChartCanvas from '../shared/chart/Canvas';
-import MarkerTooltipContents from '../shared/MarkerTooltipContents';
+import MarkerTooltipContents from '../tooltip/markers/MarkerTooltipContents';
```

Comment:
Could this just be `src/components/tooltip/Marker.js`? We already have `src/components/tooltip/CallNode.js`.

Reply:
good idea